### PR TITLE
[SC-290] Allow P-256K for secp256k1 and Allow outside EPK for encrypt_jwe.

### PR DESCRIPTION
**Pull request by windies21**

### CODE REVIEW

File: assam/jwt/jws.py [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 _verification_alg = {
     "P-256": "ES256",
-    "secp256k1": "ES256K"
+    "secp256k1": "ES256K",
+    "P-256K": "ES256K"
 }
 
 
-def encrypt_jws(signer_key: jwk.JWK, payload: dict) -> str:
+def encrypt_jws(signer_key: jwk.JWK, payload: dict, other_headers=None) -> str:
```
Comment (hyerim-kim) [inline assam/jwt/jws.py:12]: How about using `extra_header_items` instead of `other_headers`?

File: setup.py [hunk 2]
```diff
@@ -10,7 +10,11 @@
     "description": "Cryptographic library for Lite Vault servers.",
     "packages": find_packages(),
     "python_requires": ">=3.7.3",
-    "install_requires": ["jwcrypto", "iconsdk", "python-jwt"],
+    "install_requires": [
```
Comment (hyerim-kim) [inline setup.py]: Why don't you specify some required packages in `requirements.txt` file?

File: assam/jwt/jws.py (hunk 1 above)
Comment (windies21) [reply assam/jwt/jws.py:12]: That name was not made by me. It came from python-jwt.

File: setup.py (hunk 2 above)
Comment (windies21) [reply setup.py]: It seems that the person who first created this library wanted to use only setup.py.

File: assam/jwt/jws.py (hunk 1 above)
Comment (hyerim-kim) [reply assam/jwt/jws.py:12]: Even if the name came from the origin repository, it makes me be confused like there are multi headers.

Comment (windies21) [reply assam/jwt/jws.py:12]: This value is only passed to that library. A different name seems to be more confusing.

File: setup.py (hunk 2 above)
Comment (hyerim-kim) [reply setup.py]: It makes be not clear to specify the dependencies of this repository. If the person intends to use only setup.py, requirements.txt should be removed.

**Review by hyerim-kim — APPROVED** [review]

**Pull request merged**